### ISSUE TITLE
Concourse windows - Base resource type not found : git

### ISSUE THREAD

**Issue by Rahul-Bhargav** [concourse/concourse#4171]
## Bug Report

Trying to run concourse on a Windows 10 Pro (version 1809).
Base resource type git is not found.
Further investigation on the atc Postgres database lead to empty tables
`SELECT * FROM public.worker_base_resource_types, SELECT * FROM public.base_resource_types`

Maybe some inital migrations are not run? 

## Steps to Reproduce
1 . Downloaded the concourse binaries, added it to system path
2.  Installed postgres, added new user and created atc database
3. Generated keys.
4.  Added ENV variables.
env variables used - https://gist.github.com/Rahul-Bhargav/6c12a3fe5dd8fd5d1e2392049706dc22
5. set-pipeline : https://gist.github.com/Rahul-Bhargav/756a60bce5980ea7274f883a0716339e
the resource part looks like this
```---
resources:
- name: source-code-resource
  type: git
  source:
    uri: https://bitbucket.org/resilienx/fraihmwork.git
    branch: FRAIHM-213-implement-build-pipeline
    username: ((git-user-name))
    password: ((git-password))
```
6. run` concourse quickstart`

## Expected Results

Concourse runs perfect performs maven task.

## Actual Results

![image](https://user-images.githubusercontent.com/25194074/61967720-1565c800-afa4-11e9-88cf-63266a6e40a2.png)

## Version Info

* Concourse version: 5.4
* Deployment type (BOSH/Docker/binary): binary
* Infrastructure/IaaS: Windows
* Browser (if applicable):
* Did this used to work? Works when I used docker-compose with the concourse/concourse image.

**Comment by kcmannem** [comment]
I think you need at least one linux worker. Are you running a solo windows worker?

**Comment by Rahul-Bhargav** [comment]
@kcmannem Yes I'm running a solo windows worker. I now kind of realize the problem here. Is it possible to run a linux worker on windows OS? through docker or something?

**Comment by kcmannem** [comment]
I'm sure you could set something up with docker. I haven't done it before and I haven't used windows in years. But you can definitely use our docker-compose.yml file  in the root of the repo and modify it to create just a worker on docker. If you don't actually need a windows worker, then your probably better off just doing the whole deployment on docker.

**Comment by Rahul-Bhargav** [comment]
Yes, Also I just tried spinning up a worker using docker only to remember that the docker worker can not access host concourse. So just reverted back to docker-compose deployment (Not stable at all). 

I'm still running into a lot of issues with that (docker deployment). Keep seeing these `file not found`  or `unknown-handler` etc every time. Haven't seen a green build since quiet a long time. Lord knows I want to see that green. 

Thanks for the help @kcmannem .

Closing this as this is not a bug or an even an issue.
